int8 coords  # 0 = camera coords
             # 1 = 2D coords
             # 2 = 3D coords
---
bool success  # bool feedback
string msg  # information message
